### ISSUE TITLE
Uploaded media does not respond with appropriate CORS headers

### ISSUE THREAD

**Issue by richardvanbergen** [payloadcms/payload#64]
# Bug Report

I'm trying to inline an SVG uploaded to the admin using `react-svg` so that I can manipulate its properties. Unfortunately, this means making an async request to the resource and although the media has public read access, Chrome is blocking because the CORS headers in payload.config.js are not being added to the file response.

## Expected Behavior

Given the following `payload.config.js`

```
export default buildConfig({
  serverURL: 'http://localhost:3000',
  cors: ['http://localhost:3001'],
  ...
})
```

Script requests made to media such as `http://localhost:3000/media/images/bv-logo.svg` should respect the CORS setting. Like so:

```
Access-Control-Allow-Origin: http://localhost:3001
````

## Current Behavior

No access control headers are set:

![image](https://user-images.githubusercontent.com/3854592/107321424-d5aef480-6ae5-11eb-921e-201054876e6d.png)

## Steps to Reproduce

1. Create a collection with `upload: true`
2. Allow public read access
3. Upload an asset
4. Configure CORS to accept from all
5. Make an async request to the asset from another client
6. Error

**Comment by jmikrut** [comment]
Good call! Easy fix. Will be deploying a patch version shortly to address.

Thank you!

**Comment by DanRibbens** [comment]
https://github.com/payloadcms/payload/pull/66
I haven't had a chance to fully test this yet.

**Comment by DanRibbens** [comment]
I merged and published v0.2.12 that includes the fix. @richardvanbergen let us know if you have any trouble.

**Comment by richardvanbergen** [comment]
Looking good from over here!

![image](https://user-images.githubusercontent.com/3854592/107842449-8d097b00-6e06-11eb-9553-7fc18ee0ef22.png)
